luaunit.lua

```diff
@@ -753,6 +753,30 @@ function M.assertNotIsNil(value)
     end
 end
 
+function M.assertIsNaN(value)
+    if type(value) ~= "number" or value == value then
+        failure("expected: nan, actual: " ..prettystr(value), 2)
+    end
+end
+
+function M.assertNotIsNaN(value)
+    if type(value) == "number" and value ~= value then
+        failure("expected non nan value, received nan", 2)
+    end
+end
+
+function M.assertIsInf(value)
+    if type(value) ~= "number" or math.abs(value) ~= 1/0 then
+        failure("expected: inf, actual: " ..prettystr(value), 2)
```

Comment:
Don't you think that it would be better to use a constant for comparing with inf. I am uneasy with performing the calculation each time...